perfkitbenchmarker/benchmark_spec.py

```diff
@@ -110,13 +113,14 @@
 flags.DEFINE_enum('cloud', GCP, [GCP, AZURE, AWS, DIGITALOCEAN],
                   'Name of the cloud to use.')
 flags.DEFINE_enum(
-    'os_type', DEBIAN, [DEBIAN, RHEL],
+    'os_type', DEBIAN, [DEBIAN, RHEL, UBUNTU_CONTAINER],
     'The VM\'s OS type. Ubuntu\'s os_type is "debian" because it is largely '
     'built on Debian and uses the same package manager. Likewise, CentOS\'s '
     'os_type is "rhel". In general if two OS\'s use the same package manager, '
     'and are otherwise very similar, the same os_type should work on both of '
     'them.')
-
+flags.DEFINE_string('scratch_dir', '/',
```

Comment:
The default works when running with docker. This was added because '/' is read only in the specific image I'm using, so for this case I need scratch_dir=/usr/local but that's a particularity of my use case.